readme.md

```diff
@@ -1,72 +1,42 @@
 # Retrospect - Public GIT Repository #
-[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/retrospect-addon/plugin.video.retrospect)](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
 [![Build Status](https://travis-ci.com/retrospect-addon/plugin.video.retrospect.svg?branch=master)](https://travis-ci.com/retrospect-addon/plugin.video.retrospect)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=retrospect-addon:plugin.video.retrospect&metric=alert_status)](https://sonarcloud.io/dashboard?id=retrospect-addon:plugin.video.retrospect)
-
+[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![Python](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue)](https://kodi.tv/article/attention-addon-developers-migration-python-3)
```

Comment:
Not sure why this is relevant if it supports both Python 2 and 3.
Except maybe to show off ;-)

Reply:
Well, partly. But add-ons are not accepted anymore if they are not compatible with both....and I wanted a full line 😄

Reply:
Most users don't know and don't care what is and is not accepted by a repository policy.
It just needs to work for them. So that information is lost on everybody.
For people that do care, they will know if the add-on still works and is maintained.